jdbc/src/test/java/nextstep/jdbc/JdbcTemplateTest.java

```diff
@@ -1,5 +1,98 @@
+            () -> assertThat(users.get(0).checkPassword("password")).isTrue(),
+            () -> assertThat(users.get(0).getEmail()).isEqualTo("hkkang@woowahan.com"));
+    }
+
+    @Test
+    void update() {
+        // given
+        final var insertQuery = "insert into users (account, password, email) values (?, ?, ?)";
+        jdbcTemplate.update(insertQuery, "gugu", "password", "hkkang@woowahan.com");
+
+        // when
+        final var sql = "update users set account = ?, password = ?, email = ? where id = ?";
+
+        jdbcTemplate.update(sql, "gugu", "new password", "new email@woowahan.com", 1);
+
+        // then
+        final var selectQuery = "select id, account, password, email from users";
+        List<TestUser> users = jdbcTemplate.query(selectQuery, ROW_MAPPER);
+
+        assertAll(() -> assertThat(users).hasSize(1), () -> assertThat(users.get(0).getAccount()).isEqualTo("gugu"),
+            () -> assertThat(users.get(0).checkPassword("password")).isFalse(),
+            () -> assertThat(users.get(0).checkPassword("new password")).isTrue(),
+            () -> assertThat(users.get(0).getEmail()).isEqualTo("new email@woowahan.com"));
+    }
+
+    @Test
+    void queryForObject() {
+        // given
+        final var insertQuery = "insert into users (account, password, email) values (?, ?, ?)";
+        jdbcTemplate.update(insertQuery, "gugu", "password", "hkkang@woowahan.com");
+
+        // when, then
+        final var sql = "select id, account, password, email from users where account = ?";
+        TestUser user = jdbcTemplate.queryForObject(sql, ROW_MAPPER, "gugu");
+
+        assertAll(() -> assertThat(user.getAccount()).isEqualTo("gugu"),
+            () -> assertThat(user.checkPassword("password")).isTrue(),
+            () -> assertThat(user.getEmail()).isEqualTo("hkkang@woowahan.com"));
+    }
+
+    @Test
+    void query() {
+        // given
+        final var insertQuery1 = "insert into users (account, password, email) values (?, ?, ?)";
+        jdbcTemplate.update(insertQuery1, "gugu", "password", "hkkang@woowahan.com");
+
+        final var insertQuery2 = "insert into users (account, password, email) values (?, ?, ?)";
+        jdbcTemplate.update(insertQuery2, "neo", "password", "neo@woowahan.com");
+
+        // when, then
+        final var sql = "select id, account, password, email from users";
+        List<TestUser> users = jdbcTemplate.query(sql, ROW_MAPPER);
+
+        assertAll(() -> assertThat(users).hasSize(2), () -> assertThat(users.get(0).getAccount()).isEqualTo("gugu"),
+            () -> assertThat(users.get(1).getAccount()).isEqualTo("neo"),
+            () -> assertThat(users.get(0).checkPassword("password")).isTrue(),
+            () -> assertThat(users.get(1).checkPassword("password")).isTrue(),
+            () -> assertThat(users.get(0).getEmail()).isEqualTo("hkkang@woowahan.com"),
+            () -> assertThat(users.get(1).getEmail()).isEqualTo("neo@woowahan.com"));
+    }
```

Comment:
```suggestion
        assertAll(
            () -> assertThat(users).hasSize(2), 
            () -> assertThat(users.get(0).getAccount()).isEqualTo("gugu"),
            () -> assertThat(users.get(1).getAccount()).isEqualTo("neo"),
            () -> assertThat(users.get(0).checkPassword("password")).isTrue(),
            () -> assertThat(users.get(1).checkPassword("password")).isTrue(),
            () -> assertThat(users.get(0).getEmail()).isEqualTo("hkkang@woowahan.com"),
            () -> assertThat(users.get(1).getEmail()).isEqualTo("neo@woowahan.com")
        );
    }
```

위와 같이 개행하면 더 좋을 것 같아요! 판다의 스타일이시니까, 고려만 해보셔도 좋을 것 같아요~!